.changeset/slow-cheetahs-try.md

```diff
@@ -0,0 +1,7 @@
+---
+'xstate': minor
```

Comment:
I've added a changeset here - @davidkpiano  maybe you could review the wording and the level of the detail included here

Reply:
Some nits, LGTM